### ISSUE TITLE
Flash helper for 3.1

### ISSUE THREAD

**Issue by dereuromark** [FriendsOfCake/bootstrap-ui#96]
```
Error: Unsupported operand types vendor\friendsofcake\bootstrap-ui\src\View\Helper\FlashHelper.php 
```

I think the 3.1 flash messages have more nesting and therefore break with the current bootstrap plugin.

**Comment by ADmad** [comment]
A fix would be welcome.

**Comment by Modicrumb** [comment]
I fixed this in my pull request #99 
:)
